vscode/oso/Makefile

```diff
@@ -15,14 +15,15 @@ node_modules: package.json client/package.json server/package.json
 	yarn install
 	@touch $@
 
-fmtcheck: clean node_modules
+fmtcheck: clean
 	yarn fmtcheck
 
-lint: clean wasm node_modules
+lint: clean wasm
```

Comment:
I guess this was already here but why clean on every operation?

Reply:
Make sure there aren't any old compiled files hanging out — any reason not to?

Reply:
Clean = delete intermediate files that could speed up build for me. i.e. `cargo clean` torches all the intermediate state. So running it between every build was surprising to me for that reason. But I guess in this case it's just removing the compiled files so if you feel that's necessary than seems okay.

Reply:
> Clean = delete intermediate files that could speed up build for me. i.e. cargo clean torches all the intermediate state.

Isn't that going to slow your build down since you're going to have to recompile everything instead of just the incremental change since your last compilation?

> But I guess in this case it's just removing the compiled files so if you feel that's necessary than seems okay.

🤷 it doesn't really affect anything — I think TypeScript & ESBuild both support some form of incremental compilation but I'm not sure I have the knobs fiddled correctly to take advantage of that and it also doesn't really matter since compilation w/ either tool takes less than a second